prep-example/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>prep-example</title>
+</head>
+<body>
+  <h1>I would like to build</h1>
+  <p>
+    <!-- write your app ideas here 👇 -->
+
+
+
+  </p>
```

Comment:
Looks like your changes weren't saved before the `git add` and `git commit` steps. That's ok! You can come back to this one after your HTML Skeleton exercise.